warehouse/migrations/versions/7165e957cddc_create_table_for_warehouse_.py

```diff
@@ -0,0 +1,52 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+create table for warehouse administration flags
+
+Revision ID: 7165e957cddc
+Revises: 1e2ccd34f539
+Create Date: 2018-02-17 18:42:18.209572
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '7165e957cddc'
+down_revision = '1e2ccd34f539'
+
+# Note: It is VERY important to ensure that a migration does not lock for a
+#       long period of time and to ensure that each individual migration does
+#       not break compatibility with the *previous* version of the code base.
+#       This is because the migrations will be ran automatically as part of the
+#       deployment process, but while the previous version of the code is still
+#       up and running. Thus backwards incompatible changes must be broken up
+#       over multiple migrations inside of multiple pull requests in order to
+#       phase them in over multiple deploys.
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Not a big deal, but I normally remove these lines, at least as a marker that I looked at the migration and it's doing the right thing.